package.xml

```diff
@@ -1,19 +1,31 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model
+  href="http://download.ros.org/schema/package_format3.xsd"
+  schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>geolib2</name>
-  <version>0.0.0</version>
+  <version>0.1.0</version>
   <description>The geolib2 package</description>
 
-  <maintainer email="S.v.d.Dries@tue.nl">Sjoerd van den Dries</maintainer>
+  <author email="S.v.d.Dries@tue.nl">Sjoerd van den Dries</author>
+
+  <maintainer email="MatthijsBurgh@outlook.com">Matthijs van der Burgh</maintainer>
 
   <license>BSD</license>
 
   <buildtool_depend>catkin</buildtool_depend>
 
   <depend>assimp</depend>
-  <depend>boost</depend>
   <depend>code_profiler</depend>
   <depend>cv_bridge</depend>
-  <depend>tf</depend>
+  <depend>roscpp</depend>
```

Comment:
I don't think we should add a ros dependency to this. This package has nothing to do with ros, let's keep it that way.

Reply:
I think you have added some ROS logging in past PRs. I agree that it has nothing to do with ROS, we should get rid of the ROS logging in that case.

Reply:
I agree. Do you know of another way of doing logging for packages like this which do not depend on ros?

Reply:
I tried `log4cxx`, which is used by rog logging. But my simple test, breaks the formatting of the ros logs. So getting them toi work together is nog going to be easy. So I would stick to roscpp for now.